### ISSUE TITLE
Network Asset Bundles

### ISSUE THREAD

**Issue by HelloKitty** [Superviral/Pokemon-GO-App-Assets-and-Images#2]
If you're looking for the Pokemon 3D models they come from Unity3D asset bundles that the app queries for. They're stored on Google Cloud's storage but aren't accessible without authentication.

For example, this contains a Staryu or something https://storage.googleapis.com/cloud_assets_pgorelease/bundles/android/pm0126 but without the proper signature and etc the request gets rejected.

If you can figure out how to replay these requests to load the bundles then I know a guy who has written a tool that can be used to export the fully rigged FBXs from Unity3D using the bundles.

I hope for the best! Thanks for the other assets!

**Comment by TercioViejo** [comment]
Thank you

**Comment by GeReV** [comment]
I can't say for sure, since I couldn't open them, but I think the network bundles are cached.

On my device there are files which are a few hundred kilobytes each under `/storage/emulated/0/Android/data/com.nianticlabs.pokemongo/files/bundles`.

Hope this helps and I'm not incorrect.

**Comment by bleonard252** [comment]
That is correct @GeReV. However they cannot be opened in UABE. They may be protobuf'd like the GAME_MASTER.
